src/js/components/Carousel/Carousel.js

```diff
@@ -112,7 +112,10 @@ const Carousel = ({
         // eslint-disable-next-line react/no-array-index-key
         key={index}
         icon={
-          <CurrentIcon color={activeIndex === index ? iconColor : undefined} />
+          <CurrentIcon
+            color={activeIndex === index ? iconColor : undefined}
+            a11yTitle={`Slide ${index + 1}`}
```

Comment:
Would it be more descriptive to say "Go to Carousel Slide 1"?

Reply:
I like either "Go to" or "Show" as they more specifically capture the action.

```suggestion
            a11yTitle={`Show carousel slide ${index + 1}`}
```